addons/account_payment/account_payment.py

```diff
@@ -267,7 +267,10 @@ def _get_date(self, cr, uid, context=None):
 
     def _get_ml_inv_ref(self, cr, uid, ids, *a):
         res = {}
-        for id in self.browse(cr, uid, ids):
+        ctx = None
+        if len(a) >= 3 and isinstance(a[2], dict):
+            ctx = a[2]
+        for id in self.browse(cr, uid, ids, context=ctx):
```

Comment:
@StefanRijnhart the context value is also missing [here](https://github.com/EssentNovaTeam/odoo/pull/9/files#diff-de8532a4038aac8123995b33733d6391R282) and [here](https://github.com/EssentNovaTeam/odoo/pull/9/files#diff-de8532a4038aac8123995b33733d6391R291). Do you want me to fix these as well?

Reply:
Sure